tests/Sniffs/TestCase.php

```diff
@@ -162,7 +162,7 @@ protected function assertNoSniffErrorInFile(PhpCsFile $file): void
 	private function getFormattedErrorsForFile(array $errorsForFile): string
 	{
 		$message = '';
-		foreach ($errorsForFile as $line => $errorsForPossition) {
+		foreach ($errorsForFile as $line) {
```

Comment:
I think this changes the behaviour of the code - see https://3v4l.org/XTpc9

Reply:
I believe we should not look for variables at foreachs than.

/cc @kukulich